cloud.foundry.cli/src/main/java/cloud/foundry/cli/operations/ServicesOperations.java

```diff
@@ -18,24 +23,61 @@ public ServicesOperations(DefaultCloudFoundryOperations cloudFoundryOperations)
         super(cloudFoundryOperations);
     }
 
-
-    public List<ServiceInstanceSummaryBean> getAll() {
+    /**
+     * @return all service instances in the space
+     */
+    public List<ServiceBean> getAll() {
         List<ServiceInstanceSummary> services = this.cloudFoundryOperations
                 .services()
                 .listInstances()
                 .collectList()
                 .block();
 
+        if (services == null) {
+            services = new LinkedList<>();
+        }
 
         // create a list of special bean data objects, as the summaries cannot be serialized directly
-        List<ServiceInstanceSummaryBean> beans = new ArrayList<>();
+        List<ServiceBean> beans = new ArrayList<>(services.size());
         for (ServiceInstanceSummary serviceInstanceSummary : services) {
-            beans.add(new ServiceInstanceSummaryBean(serviceInstanceSummary));
+            beans.add(new ServiceBean(serviceInstanceSummary));
         }
-        // create YAML document
-        Yaml yaml = YamlCreator.createDefaultYamlProcessor();
 
-        return yaml.loadAs(yaml.dump(beans), List.class);
+        return beans;
+    }
+
+    /**
+     * Creates a new service in the space and binds apps to it. In case of an error, the creation- and binding-process
+     * is discontinued.
+     * @param serviceBean serves as template for the service to create
+     * @throws CreationException when the creation was not successful
+     */
+    public void create(ServiceBean serviceBean) throws CreationException {
```

Comment:
This whole method is way too packed. I would recommend adding a few blank lines to group "logical blocks".